authentication-demo-service/pom.xml

```diff
@@ -251,7 +229,7 @@
 		<dependency>
 			<groupId>io.mosip.kernel</groupId>
 			<artifactId>kernel-keymanager-service</artifactId>
-			<version>1.2.0.1-SNAPSHOT</version>
+			<version>1.2.1-java21-SNAPSHOT</version>

```

Comment:
If it is develop branch then version should be 1.2.1-SNAPSHOT

Reply:
Done.